docker-compose-dev.yml

```diff
@@ -41,24 +45,25 @@ services:
     image: cernopendata/static # Use this to make sure that COD3 Python-code image is built only once.
     depends_on:
       - static # Just to make sure that latest cernopendata/static -image is build before creating this container.
-    command: "cernopendata run -h 0.0.0.0 --reload" # In case one needs to test/work with uWSGI comment out this CMD override.
+    command: bash -c "cernopendata run -h 0.0.0.0 --reload" # In case one needs to test/work with uWSGI comment out this CMD override.
     environment:
       - DEBUG=True
+      - FLASK_DEBUG=1
       - WDB_SOCKET_SERVER=wdb
       - WDB_NO_BROWSER_AUTO_OPEN=True
-      - APP_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://cernopendata:dbpass123@postgresql:5432/cernopendata
-      - APP_CACHE_REDIS_HOST=redis
-      - APP_CACHE_REDIS_URL=redis://redis:6379/0
-      - APP_ACCOUNTS_SESSION_REDIS_URL=redis://redis:6379/1
-      - APP_BROKER_URL=amqp://guest:guest@rabbitmq:5672/ # Celery 3
-      - APP_CELERY_BROKER_URL=amqp://guest:guest@rabbitmq:5672/ # Celery 4
-      - APP_CELERY_RESULT_BACKEND=redis://redis:6379/2
-      - APP_SEARCH_ELASTIC_HOSTS=elasticsearch
-      - APP_PIDSTORE_DATACITE_TESTMODE=False
-      - APP_PIDSTORE_DATACITE_DOI_PREFIX=10.5072
-      - APP_PIDSTORE_DATACITE_USERNAME=CERN.OPENDATA
-      - APP_PIDSTORE_DATACITE_PASSWORD=CHANGE_ME
-      - APP_PIDSTORE_LANDING_BASE_URL=http://opendata.cern.ch/record/
+      - INVENIO_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://cernopendata:dbpass123@postgresql:5432/cernopendata
+      - INVENIO_CACHE_REDIS_HOST=redis
+      - INVENIO_CACHE_REDIS_URL=redis://redis:6379/0
+      - INVENIO_ACCOUNTS_SESSION_REDIS_URL=redis://redis:6379/1
+      - INVENIO_BROKER_URL=amqp://guest:guest@rabbitmq:5672/ # Celery 3
+      - INVENIO_CELERY_BROKER_URL=amqp://guest:guest@rabbitmq:5672/ # Celery 4
+      - INVENIO_CELERY_RESULT_BACKEND=redis://redis:6379/2
```

Comment:
Getting warning messages about upgrade path to Celery 6.X.Y:
```console
worker_1         | /opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/celery/app/utils.py:206: CDeprecationWarning: 
worker_1         |     The 'CELERY_RESULT_BACKEND' setting is deprecated and scheduled for removal in
worker_1         |     version 6.0.0. Use the result_backend instead
worker_1         | /opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/celery/app/utils.py:206: CDeprecationWarning: 
worker_1         |     The 'CELERY_TASK_SERIALIZER' setting is deprecated and scheduled for removal in
worker_1         |     version 6.0.0. Use the task_serializer instead
worker_1         | /opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/celery/app/utils.py:206: CDeprecationWarning: 
worker_1         |     The 'CELERY_ACCEPT_CONTENT' setting is deprecated and scheduled for removal in
worker_1         |     version 6.0.0. Use the accept_content instead
worker_1         | /opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/celery/app/utils.py:206: CDeprecationWarning: 
worker_1         |     The 'BROKER_URL' setting is deprecated and scheduled for removal in
worker_1         |     version 6.0.0. Use the broker_url instead
```

Reply:
This should be handled by [Invenio-Celery](https://github.com/inveniosoftware/invenio-celery/blob/6186820328aa2e1a70565f31beab7a19f8700092/invenio_celery/config.py). Probably not happening any time soon, see [Celery version pinning](https://github.com/inveniosoftware/invenio-celery/blob/6186820328aa2e1a70565f31beab7a19f8700092/setup.py#L45).